cfgmgr/vlanmgr.cpp

```diff
@@ -50,19 +50,19 @@ VlanMgr::VlanMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, c
     // Initialize Linux dot1q bridge and enable vlan filtering
     // The command should be generated as:
     // /bin/bash -c "/sbin/ip link del Bridge 2>/dev/null ;
-    //               /sbin/ip link add Bridge up type bridge &&
+    //               /sbin/ip link add Bridge up mtu 9100 type bridge &&
     //               /sbin/bridge vlan del vid 1 dev Bridge self;
     //               /sbin/ip link del dummy 2>/dev/null;
-    //               /sbin/ip link add dummy type dummy &&
+    //               /sbin/ip link add dummy mtu 9100 type dummy &&
     //               sbin/ip link set dummy master Bridge"
 
     const std::string cmds = std::string("")
       + BASH_CMD + " -c \""
       + IP_CMD + " link del " + DOT1Q_BRIDGE_NAME + " 2>/dev/null; "
-      + IP_CMD + " link add " + DOT1Q_BRIDGE_NAME + " up type bridge && "
+      + IP_CMD + " link add " + DOT1Q_BRIDGE_NAME + " up mtu " + DEFAULT_MTU_STR + " type bridge && "
```

Comment:
Does kernel changes this configured Vlan interface MTU when you add another interface to Vlan with a lower MTU value?